src/specialized/pclmulqdq.rs

```diff
@@ -51,6 +57,7 @@ const K6: i64 = 0x1db710640;
 const P_X: i64 = 0x1DB710641;
 const U_PRIME: i64 = 0x1F7011641;
 
+#[cfg(feature = "use_std")]
```

Comment:
This looks like a typo, should this be `#[cfg(feature = "std")]` instead? (same on the other definition below)